### PR TITLE
preformance mode

### PR DESCRIPTION
**Pull request by holiber**

### CODE REVIEW

File: app/components/MixerVolmeter.vue [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+</div>
+
+</template>
+
+<script lang="ts" src="./MixerVolmeter.vue.ts"></script>
+
+<style lang="less" scoped>
+@import "../styles/index";
+
+.volmeter {
```
Comment (andycreeth) [inline app/components/MixerVolmeter.vue]: It looks like this CSS was copied from `MixerItem.vue`.  Can we remove from there if this is its new home?

File: app/services/customization.ts [hunk 2]
```diff
@@ -10,6 +11,7 @@ interface ICustomizationServiceState {
   livedockCollapsed: boolean;
   previewSize: number;
   livedockSize: number;
+  previewEnabled: boolean;
```
Comment (andycreeth) [inline app/services/customization.ts]: Can we call this performance mode instead?

File: app/util/menus/EditMenu.ts [hunk 3]
```diff
@@ -145,6 +147,17 @@ export class EditMenu extends Menu {
         label: 'Unlock all sources',
         click: () => this.scenesService.setLockOnAllScenes(false)
       });
+
+      this.append({ type: 'separator' });
+
+      this.append({
+        label: 'Enable Preview',
```
Comment (andycreeth) [inline app/util/menus/EditMenu.ts]: I think we should call this performance mode.

File: app/services/customization.ts [hunk 4]
```diff
@@ -28,16 +30,20 @@ export class CustomizationService extends PersistentStatefulService<ICustomizati
...
+  settingsChanged = new Subject<Partial<ICustomizationSettings>>();
+
   init() {
     super.init();
     this.setLiveDockCollapsed(true);// livedock is always collapsed on app start
   }
 
   setSettings(settingsPatch: Partial<ICustomizationSettings>) {
     this.SET_SETTINGS(settingsPatch);
+    this.settingsChanged.next(settingsPatch);
```
Comment (andycreeth) [inline app/services/customization.ts]: Does anything subscribe to this?  What is this for?

File: app/components/MixerVolmeter.vue (hunk 1 above)
Comment (holiber) [reply app/components/MixerVolmeter.vue]: 👍 +1

File: app/services/customization.ts (hunk 4 above)
Comment (holiber) [reply app/services/customization.ts]: Ups, my first attempt was event-based. Forgot to delete this.

**Comment by andycreeth** [comment]
I'm going to merge this so we can get it into the preview build I am about to release.

**Pull request merged**